tutorial/10_key_features/005_visualization.py

```diff
@@ -150,23 +88,23 @@ def objective(trial):
 
 ###################################################################################################
 # Select parameters to visualize.
-plot_parallel_coordinate(study, params=["lr_init", "n_units_l0"])
+plot_parallel_coordinate(study, params=["bagging_freq", "bagging_fraction"])
 
 ###################################################################################################
 # Visualize hyperparameter relationships. See :func:`~optuna.visualization.plot_contour` for the details.
 plot_contour(study)
 
 ###################################################################################################
 # Select parameters to visualize.
-plot_contour(study, params=["n_units_l0", "n_units_l1"])
+plot_contour(study, params=["bagging_freq", "bagging_fraction"])
 
 ###################################################################################################
 # Visualize individual hyperparameters as slice plot. See :func:`~optuna.visualization.plot_slice` for the details.
 plot_slice(study)
```

Comment:
How about specifying parameters here as well? The rendered plot looks a bit broken when it stretches outside the border https://64524-122299416-gh.circle-artifacts.com/0/docs/build/html/tutorial/10_key_features/005_visualization.html#sphx-glr-tutorial-10-key-features-005-visualization-py.